mlflow/store/runs_artifact_repo.py

```diff
@@ -88,7 +88,7 @@ def list_artifacts(self, path):
 
         :return: List of artifacts as FileInfo listed directly under path.
         """
-        self.repo.list_artifacts(path)
+        return self.repo.list_artifacts(path)
```

Comment:
this fix is in https://github.com/mlflow/mlflow/pull/1175 so this PR doesn't have to block other PRs changing APIs with ``model_uri``